client/state/login/selectors.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Retrieve the user ID for the two factor authentication process.
+ * Returns null if there is no such information yet, or user does not have 2FA enabled.
+ *
+ * @param  {Object}   state  Global state tree
+ * @return {?Number}         The user ID.
+ */
+export const getTwoFactorAuthId = ( state ) => {
+	return get( state, [ 'login', 'twoFactorAuth', 'two_step_id' ], null );
```

Comment:
Why did you prefer using an array for describing the path to this value instead of a string here and below? The following looks much shorter and readable to me:

```javascript
return get( state, 'login.twoFactorAuth.two_step_id', null );
```